<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VerificationPreferences</class>
 <widget class="QWidget" name="VerificationPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Checksums</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Automatic verification:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KButtonGroup" name="kcfg_ChecksumStrength">
        <property name="title">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="radioButton_4">
           <property name="text">
            <string>Weak (fastest)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_5">
           <property name="text">
            <string>Strong (recommened)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_6">
           <property name="text">
            <string>Strongest (slowest)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="kcfg_ChecksumAutomaticVerification">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Used checksum:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="signatureGroup">
     <property name="title">
      <string>Signature</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="kcfg_SignatureAutomaticVerification">
        <property name="text">
         <string>Automatic verification</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_SignatureAutomaticDownloading">
        <property name="text">
         <string>Automatic downloading of missing keys</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Keyservers:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QListView" name="keyServers"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="KPushButton" name="add"/>
          </item>
          <item>
           <widget class="KPushButton" name="remove"/>
          </item>
          <item>
           <widget class="KPushButton" name="up">
            <property name="text">
             <string>Move &amp;Up</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="KPushButton" name="down">
            <property name="text">
             <string>Move &amp;Down</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
     <zorder>kcfg_SignatureAutomaticVerification</zorder>
     <zorder>kcfg_SignatureAutomaticDownloading</zorder>
     <zorder>verticalSpacer</zorder>
     <zorder>label</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KButtonGroup</class>
   <extends>QGroupBox</extends>
   <header>kbuttongroup.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
